climada/engine/forecast.py

```diff
@@ -372,7 +377,8 @@ def plot_imp_map(
             "run_start": (
                 run_datetime.strftime("%d.%m.%Y %HUTC +") + lead_time_str + "d"
             ),
-            "explain_text": ("mean building damage caused by wind"),
+            "explain_text": ("mean building damage caused by " +
+                             "wind") if explain_str is None else explain_str,
```

Comment:
No need to separate "wind", except you plan to make it a variable at some point?

Reply:
I changed the line, which makes it more human readable. Now it is slightly too long for pylint, but I think readability comes first.

I also adapted the other (similar) line and the variable names containing the word "wind"